src/vanilla/store.ts

```diff
@@ -260,6 +264,7 @@ type StoreArgs = readonly [
     atom: WritableAtom<Value, Args, Result>,
     setAtom: (...args: Args) => Result,
   ) => OnUnmount | void,
+  atomOnInit: <Value>(atom: Atom<Value>, store: Store) => void,
```

Comment:
Should Init come before Mount in StoreArgs?

Reply:
I'm fine with it.